src/components/graph/graph.builder.js

```diff
@@ -49,11 +49,11 @@ function _getNodeOpacity(node, highlightedNode, highlightedLink, config) {
  * @param  {Function[]} linkCallbacks - same as {@link #graphrenderer|linkCallbacks in renderGraph}.
  * @param  {string} highlightedNode - same as {@link #graphrenderer|highlightedNode in renderGraph}.
  * @param  {Object} highlightedLink - same as {@link #graphrenderer|highlightedLink in renderGraph}.
- * @param  {number} transform - value that indicates the amount of zoom transformation.
+ * @param  {number} scaleFactor - value that indicates the amount of zoom scaleFactor.
  * @returns {Object} returns an object that aggregates all props for creating respective Link component instance.
  * @memberof Graph/builder
  */
-function buildLinkProps(link, nodes, links, config, linkCallbacks, highlightedNode, highlightedLink, transform) {
+function buildLinkProps(link, nodes, links, config, linkCallbacks, highlightedNode, highlightedLink, scaleFactor) {
```

Comment:
Renaming `transform` to `scaleFactor` in a few spots to differentiate between the full `transform` object and just the `k` value of it.